src/tests/msw-handlers/index.ts

```diff
@@ -158,8 +159,11 @@ export const registerGetDistributionListHandler = (
 		displayName?: string;
 		owners?: Array<{ id?: string; name?: string }>;
 		description?: string;
+		canRequireMembers?: boolean;
+		isOwner?: boolean;
 	},
-	error?: string
+	error?: string,
+	zimbraHideParam: ZimbraHideInGalType = 'FALSE'
```

Comment:
I would prefer to have this field inside the first argument. Consider the first argument the object with all the fields required to build the response, and not the real application dl.